include/rwahs/tasks/build

```diff
@@ -7,11 +7,102 @@ set -e
 set -u
 
 function build_showHelp {
-    echo "TASK DESCRIPTION:"
-    echo "    Create a new release, which is a tag in the code repository."
+    local programName toolsDir
+
+    # Function arguments
+    programName="${1}"
+    toolsDir="${2}"
+    shift 2
+
+    echo
+    echo "Create a new release, which is a tag in the code repository."
+    echo
+    echo "USAGE:"
+    echo "    ${programName} build -c <component> [-t <target-commitish>] [-r <release>]"
+    echo
+    echo "WHERE:"
+    echo "    -t <target-commitish> gives the branch or commit that should be released; if omitted, the default branch"
+    echo "         according to the component configuration (not the default branch configured in github) will be used."
+    echo "    -c <component> gives the name of the component to process."
+    echo "         Valid values: $(ls -m "${toolsDir}/config/${programName}/components")"
+    echo "    -r <release> gives the optional release tag override; if omitted, a value will be generated and output."
     echo
 }
 
 function build_runTask {
-    echo "Running 'build' task..."
+    local programName toolsDir ghApiRoot component releaseName releaseDate targetCommitish apiJsonTemplate apiJson
```

Comment:
This is the style I will use consistently from now.  Declare `local` variables on the first line(s), then set the function arguments from `${1}` etc into named variables (and never use the positional parameters after that), then set default values for other variables.